karate-grpc-core/src/main/java/com/github/thinkerou/karate/GrpcClient.java

```diff
@@ -116,8 +116,8 @@ protected static void log(String message, ScenarioBridge scenarioBridge) {
         log.info(message);
     }
 
-    protected String invokeCall(String name, String payload) {
-        return callIns.invoke(name, payload);
+    protected String invokeCall(String name, String payload, ScenarioBridge scenarioBridge) {
```

Comment:
persist `invokeCall(name, payload)`, and add `invokeCall(name, payload, scenarioBridge)`?